Managers/SettingsMan.h

```diff
@@ -244,6 +244,12 @@ namespace RTE {
 		/// </summary>
 		/// <param name="enable">Whether to enable smart BuyMenu navigation or not.</param>
 		void SetSmartBuyMenuNavigation(bool enable) { m_EnableSmartBuyMenuNavigation = enable; }
+
+		/// <summary>
+		/// Gets the maximum height of a column of scrap terrain to collapse, when the bottom pixel is knocked loose.
+		/// </summary>
+		/// <returns>The compacting height of scrap terrain.</returns>
+		int GetCompactingHeight() const { return m_CompactingHeight; }
```

Comment:
Agreed. I don't see this as particularly useful otherwise, as gamemodes wouldn't be able to take advantage of it. It's just a vague switch in the settings otherwise.

Actually, I have some concerns about the number of vague switches that are going into the ini. If we want modifiable values for testing purposes without recompilation, I suggest we use another system instead of throwing more-and-more obscure things into the Settings.ini. Most places I've worked have had the concept of "Tweakables", where you just define a macro somewhere and it lets it be tweaked and modified during runtime. In release builds these were typically compiled into a static value for optimization purposes, but we don't need to necessarily do that here.

(On that note, while I'm rambling - EnableCrabBombs and CrabBombThreshold being in the settings really annoys me. It's super vague, and there's absolutely zero functional utility in being able to set the crab-bomb threshol. I'd prefer that this was just a Global Script instead.)

Reply:
To me the compacting height is such a niche setting that it doesn't deserve a settings menu switch or global script. I originally made it in request of someone wanting scrap debris to behave like regular terrain (scrap setting = 0), as it used to in old school CC. 

Btw, the reason it's not a bool property is because 25 is a bullshit arbitrary value, and the setting has potential for activity use, such as making big collapsing towers of scrap terrain (or maybe someone would like to set it really high, I dunno). I will have to expose it to Lua though.